### ISSUE TITLE
[3.3.5] Bug with "Blink".

### ISSUE THREAD

**Issue by stas730** [TrinityCore/TrinityCore#17292]
**Description**:
Blink can be used to walk through the walls.
**Current behaviour**: Tell us what happens
If we going to the wall, we can use "Blink" and jump and go through the wall.
**Expected behaviour**: Tell us what should happen instead
Blink MUST NOT allow jump through the walls. Read description.
**Steps to reproduce the problem**:

1. Create Mage character. Or just learn "Blink"
2. Go to the walls (for example, smallest box in the ScottTest) 
3. Jump and Blink

**Branch(es)**: 3.3.5 / 6.x (Select the branch(es) affected by this issue) 
Maybe, all.
**TC hash/commit**:  
08c6b2a
**TDB version**:  
Maybe, all.
**Operating system**:  
All.

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing c++ create a Pull Request)

**Comment by msoky** [comment]
did you try https://github.com/TrinityCore/TrinityCore/commits/blink ?

**Comment by stas730** [comment]
Not. :(

**Comment by stas730** [comment]
No*

**Comment by msoky** [comment]
that will be new blink system...if anyone have time to finish that

**Comment by stas730** [comment]
Now I don't playing WoW because I waiting source code leak/open-source WoW.

**Comment by stas730** [comment]
waiting for*

**Comment by Keader** [comment]
i'm using kvipka blink system, and improve alot spell function.
But still have issues, like fall of map

**Comment by Goatform** [comment]
Yeah, I can confirm that too. I'm using Kvipka's blink system and it fixed a lot of things but thereare still some bugs left... Example, in gurubashi area I am able to blink through the wall in one place... but I was able to reproduce it just on that 1 small place.

**Comment by msoky** [comment]
if is that better and less buged than blink which is actually used, then maybe someone can push new blink system and fix minor bugs later..

**Comment by kvipka** [comment]
https://github.com/TrinityCore/TrinityCore/pull/16688

**Comment by kvipka** [comment]
@Goatform , i will not inspect smthing without xyz coords.

**Comment by Jinnaix** [comment]
I couldnt find any wall where under normal circumstances a player would be able to blink trough.
If you dont use .cheat cooldown, its impossible to even blink out of the test box. (.tele test and fly up inside the small box)

But it is possible to jump-blink yourself trough gameobjects - the forcefield in diremaul for example (.go boss immo). Not that this opens much of abusive potential.
